lib/puppet-strings/yard/parsers/puppet/parser.rb

```diff
@@ -22,10 +22,12 @@ def initialize(source, filename)
   # @return [void]
   def parse
     begin
-      Puppet[:tasks] = true if Puppet.settings.include?(:tasks)
-      if Puppet::Util::Package.versioncmp(Puppet.version, "5.0.0") < 0 && @file.to_s.match(/^plans\//)
-        log.warn "Skipping #{@file}: Puppet Plans require Puppet 5 or greater."
-        return
+      if @file.to_s.match(/^plans\//)
```

Comment:
@scotje this regex fails during testing because it's in the full fixtures directory. If I remove the start-of-string anchor, tests pass, but that's probably not a good solution. Any other ideas?

Reply:
We should probably extract this test into a helper method and then we can stub that out in the tests